src/main/app/case/case-api.ts

```diff
@@ -34,6 +34,25 @@ export class CaseApi {
     return this.apiClient.getCaseById(caseId);
   }
 
+  public async getExistingAndNewUserCases(
+    email: string,
+    serviceType: string,
+    logger: LoggerInstance
+  ): Promise<{ existingUserCase: CaseWithId | false; newInviteUserCase: CaseWithId | false }> {
+    const existingUserCase: CaseWithId | false = await this.getExistingUserCase(serviceType);
+    const newInviteUserCase = await this.getNewInviteCase(email, serviceType, logger);
+
+    if (process.env.ENV_TYPE !== 'PROD') {
```

Comment:
not sure if it matters but I think I removed these variables everywhere but prod

Reply:
I've switched to a more specific variable